tests/acm_observability/test_acm_observability.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from const import HUB_CLUSTER
+
+
+pytestmark = pytest.mark.acm_observability
+
+
+class TestACMObservability:
+    def test_hub_etcd_metrics_exist(
+        self,
+        clusters_etcd_metrics,
+    ):
+        latest_hub_etcd_db_size = clusters_etcd_metrics[HUB_CLUSTER][-1]
+        assert (
+            isinstance(latest_hub_etcd_db_size, float)
+            and latest_hub_etcd_db_size >= 0.0
```

Comment:
No need to set `0.0`, `0` will do the job

Is `0` is excepted value?

Reply:
No, etcd holds data regarding cluster components, therefore if etcd db size is 0 then it must be corrupted.
I'll update the code to `>` rather than `>=`

Reply:
done